tools/transformer/src/transformations/digest.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Remove digest auth from paths and security schemes
+ */
+function applyDigestTransformations(api) {
+  const hasSchemas = api && api.components && api.components.securitySchemes;
+  if (hasSchemas) {
+    delete api.components.securitySchemes.DigestAuth;
+    Object.keys(api.paths).forEach((path) => {
+      Object.keys(api.paths[path]).forEach((method) => {
+        delete api.paths[path][method].security;
+      });
+    });
+  }
+
+  return api;
+}
+
+module.exports = {
+  applyDigestTransformations,
+};
```

Comment:
>  also, have you considered adding this filter to the OpenAPI generation side of things?

If you feel that this will make sense we can do it.
Right now digest is printed as auth mode in the docs so this is more of generated source code problem.

Reply:
From my perspective, we should always remove security provided by openapi generator (do not work with autogenerated oauth as it will not ensure proper security updates.
Libraries are part of the generator templates - they cannot be detected by dependabot etc.
If we were to get any other auth it will be always supplied externally. That is why I have made a conscious decision to remove all security for sdks (even future ones).

Reply:
>we should always remove security provided by openapi generator (do not work with auto generated oauth as it will not ensure proper security updates.

If this is your recommendation then updating to clarify we don't want security in general seems good to me


>If you feel that this will make sense we can do it.

I think based on above that generated code should never used I feel more strongly on doing on the server as a new filter option to remove security